automatic/composer/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+# <img src="https://rawcdn.githack.com/composer/windows-setup/80f687d4ec2c5a199770c131525fd08ff93d0366/chocolatey/composer-logo.png" width="48" height="48"/> [composer](https://chocolatey.org/packages/composer)
+
+Composer Setup downloads and installs the latest version of Composer, the PHP Dependency Manager, so you can use it easily from the command line: just type `composer` from any location to use it.
+
```

Comment:
Please add a `## Features` section in this description.
This can just list the different features that are available for this software.

Reply:
Done